network-metrics-nuttcp.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
It looks like this script is in the wrong location - shouldn't it live below `tests/metrics/`?